<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewAssetPage</class>
 <widget class="QDialog" name="NewAssetPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QHBoxLayout" name="payToLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QValidatedLineEdit" name="payTo">
             <property name="toolTip">
              <string>The Crown address to send the payment to</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="addressBookButton">
             <property name="toolTip">
              <string>Choose previously used address</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../crown.qrc">
               <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="shortcut">
              <string>Alt+A</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="pasteButton">
             <property name="toolTip">
              <string>Paste address from clipboard</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../crown.qrc">
               <normaloff>:/icons/editpaste</normaloff>:/icons/editpaste</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="shortcut">
              <string>Alt+P</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="deleteButton">
             <property name="toolTip">
              <string>Remove this entry</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../crown.qrc">
               <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QComboBox" name="typecomboBox">
           <property name="toolTip">
            <string/>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::NoInsert</enum>
           </property>
           <item>
            <property name="text">
             <string>Token</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Unique</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Equity</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Points</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Credits</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="expiry">
           <property name="toolTip">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLineEdit" name="name">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Asset Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="symbol">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Asset Symbol</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="inputAmount">
           <property name="toolTip">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string extracomment="Input Amount">Input Amount (0.00) CRW</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="outputAmount">
           <property name="toolTip">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string extracomment="Output Amount">Issuing/Output (0.00)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="convertcheckBox">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Convertable</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="transfercheckBox">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Transferable</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="restrictedcheckBox">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Restricted</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="limitedcheckBox">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Limited</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="divisiblecheckBox">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Divisible</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="contracturl">
       <property name="placeholderText">
        <string>Contract url</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="scriptcode">
       <property name="placeholderText">
        <string>Contract Script</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="nftTextEdit">
       <property name="placeholderText">
        <string>Enter Data Here</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Create">
         <property name="toolTip">
          <string extracomment="Create the Asset"/>
         </property>
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qt/qvalidatedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../crown.qrc"/>
 </resources>
 <connections/>
</ui>
